ChatQnA/docker/gaudi/compose.yaml

```diff
@@ -39,7 +39,7 @@ services:
       no_proxy: ${no_proxy}
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
-      HABANA_VISIBLE_DEVICES: all
+      HABANA_VISIBLE_DEVICES: ${HABANA_VISIBLE_DEVICES}
```

Comment:
We need to specify different variables and different devices for line 42 and 124 as both are different applications.,